tests/apps/service_invocation/go.sum

```diff
@@ -153,6 +153,7 @@ github.com/dapr/components-contrib v0.4.1-0.20201015165349-4e77594a4d0f/go.mod h
 github.com/dapr/dapr v0.4.1-0.20200228055659-71892bc0111e/go.mod h1:c60DJ9TdSdpbLjgqP55A5u4ZCYChFwa9UGYIXd9pmm4=
 github.com/dapr/dapr v0.11.1-0.20201016010930-cf4e080c1a54 h1:Xheq+z2YeR/VeeMmaJMKZAJKmiwWwnA6Lz8rh7uFvmQ=
 github.com/dapr/dapr v0.11.1-0.20201016010930-cf4e080c1a54/go.mod h1:b2aBjSSdOBZ6TeW5H6GVZL6eIgiqym7d2JvYWkWOQAc=
+github.com/dapr/dapr v0.11.3 h1:FYqbIcGpH1iCOs8a7f0sOEAXRrY6Iywqc4FS3tylsFY=
```

Comment:
Run this file https://github.com/dapr/dapr/blob/master/tests/update_testapps_dependencies.sh

Reply:
Done.